### PR TITLE
Apply reorder-python-imports to all files

### LINKED ISSUE #4233: Import reordering pre-commit hook
Let's introduce a hook for us that orders imports.  Inspired by a recent implicit conflict caused by an import reordering + an import removal. https://github.com/pytest-dev/pytest/pull/4028#issuecomment-432842271

There's basically two options here:

1. [`seed-isort-config`](https://github.com/asottile/seed-isort-config) + [`isort`](https://github.com/pre-commit/mirrors-isort)

there's some configuration that makes black happy that's necessary

2. [`reorder-python-imports`](https://github.com/asottile/reorder_python_imports)

no configuration necessary, the only options are compatible with `black`

### PR DESCRIPTION
**Pull request by asottile**
ah shoot, I should write a changelog entry -- maybe tomorrow

Resolves #4233

### CODE REVIEW

File: src/_pytest/_argcomplete.py [hunk 1]
```diff
@@ -53,9 +53,12 @@
   which should throw a KeyError: 'COMPLINE' (which is properly set by the
   global argcomplete script).
 """
-from __future__ import absolute_import, division, print_function
-import sys
+from __future__ import absolute_import
```
Comment (RonnyPfannschmidt) [inline src/_pytest/_argcomplete.py]: @asottile can you change the tool so for __future__ imports it does one line?

Comment (asottile) [reply src/_pytest/_argcomplete.py]: nope, not configurable intentionally

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -23,6 +23,11 @@ repos:
         language_version: python3
     -   id: flake8
         language_version: python3
+-   repo: https://github.com/asottile/reorder_python_imports
+    rev: v1.3.2
+    hooks:
+    -   id: reorder-python-imports
+        args: ['--application-directories=.:src']
```
Comment (asottile) [inline .pre-commit-config.yaml]: I should probably also set `--add-import 'from __future__ import absolute_import'` and/or the others we want to enforce on all files

File: src/_pytest/_argcomplete.py (hunk 1 above)
Comment (RonnyPfannschmidt) [reply src/_pytest/_argcomplete.py]: @asottile not asking for a configuration there ^^

**Pull request merged**